### PR TITLE
feat: avatar 1st commit

### PR DESCRIPTION
**Pull request by ankit9015**
Feat consists:
1. Circle avatar (default)
2. Size variation for avatar -sm -md -l
3. Square avatar
4. Avatar for icons and texts

Will update later: To create this example I used avatar.css directly for avatar.html. I will later connect this to component css when I start working on documentation and do the necessary edits.
avatar

### CODE REVIEW

**Review by paarthbajaj — COMMENTED** [review]
Everything looks good. No suggestions from my side. Ready to merge.

File: avatar.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
+html {
+    font-size: 62.5%;
+}
+
+.avatar {
+    border-radius: 50%;
+    
+    width: 3ex;      /*width and height of the avatara is controlled by font-size in -sm -md -l*/
+    height: 3ex;
```
Comment (SAYUK09) [inline avatar.css:9]: can we use `em` instead of `ex` if possible? We can explore a bit.

File: avatar.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+.avatar-sm {
+    font-size: 3rem;
+}
+
+.avatar-md {
+    font-size: 4rem;
+}
+
+.avatar-l {
```
Comment (SAYUK09) [inline avatar.css:22]: We can use `lg` instead of `l` as it is more standard

File: avatar.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    <title>SocketUI | Components | Avatar</title>
+
+    <script src="https://kit.fontawesome.com/124ff44aed.js" crossorigin="anonymous"></script>
+
+    <link rel="stylesheet" href="./avatar.css">
+</head>
+<body>
+    <img class="avatar avatar-sm" src="https://picsum.photos/id/1/200/300" alt="">
+    <img class="avatar avatar-md " src="https://picsum.photos/id/1/200/300" alt="">
+    <img class="avatar avatar-l " src="https://picsum.photos/id/1/200/300" alt="">
```
Comment (SAYUK09) [inline avatar.html:16]: Don't leave the `alt` tag empty for accessibility purposes.

File: avatar.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SocketUI | Components | Avatar</title>
```
Comment (SAYUK09) [inline avatar.html:7]: We can reverse the order. Start with **Avatar** followed by **Components**

**Review by SAYUK09 — CHANGES_REQUESTED** [review]
Clean code. Here are a few suggestions you can think about.

**Review by paarthbajaj — APPROVED** [review]

**Pull request merged**